addons/hearing/XEH_postInit.sqf

```diff
@@ -100,6 +91,12 @@ GVAR(lastPlayerVehicle) = objNull;
         call FUNC(updateHearingProtection);
     }, true] call CBA_fnc_addPlayerEventHandler;
 
+    addMissionEventHandler ["ProjectileCreated", {
+        params ["_projectile"];
+
+        _projectile addEventHandler ["Explode", {call FUNC(explosion)}];
```

Comment:
strange locality issues with this (local 2 client MP)
server player's grenade and rockets don't trigger on the 2nd client
client's grenade does trigger on server but not the rocket

Server (q1):
```
Fired: [q1,"Throw","HandGrenadeMuzzle","HandGrenadeMuzzle","GrenadeHand","HandGrenade",164107: handgrenade_throw.p3d,q1]
Explode: [164107: handgrenade_throw.p3d,[1726.54,5510.9,5.53363],[0.0211423,-0.154011,0.0338565]]

Fired: [q1,"launch_RPG32_F","launch_RPG32_F","Single","R_PG32V_F","RPG32_F",164199: pg32v_rocket.p3d,q1]
Explode: [164199: pg32v_rocket.p3d,[1902.91,5710.63,19.3332],[134.031,108.958,4.30392]]

Fired: [q2,"launch_RPG32_F","launch_RPG32_F","Single","R_PG32V_F","RPG32_F",164203: pg32v_rocket.p3d,q2]

Fired: [q2,"Throw","HandGrenadeMuzzle","HandGrenadeMuzzle","GrenadeHand","HandGrenade",164205: handgrenade_throw.p3d,q2]
Explode: [164205: handgrenade_throw.p3d,[1718.96,5509.69,5.53045],[0.350581,-0.363032,-0.56609]]
```


Client (q2):
```
Fired: [q1,"Throw","HandGrenadeMuzzle","HandGrenadeMuzzle","GrenadeHand","HandGrenade",164212: handgrenade_throw.p3d,q1]

Fired: [q1,"launch_RPG32_F","launch_RPG32_F","Single","R_PG32V_F","RPG32_F",164389: pg32v_rocket.p3d,q1]

Fired: [q2,"launch_RPG32_F","launch_RPG32_F","Single","R_PG32V_F","RPG32_F",164450: pg32v_rocket.p3d,q2]
Explode: [164450: pg32v_rocket.p3d,[1998.34,5643.82,9.89063],[163.408,52.7211,-2.27858]]

Fired: [q2,"Throw","HandGrenadeMuzzle","HandGrenadeMuzzle","GrenadeHand","HandGrenade",164462: handgrenade_throw.p3d,q2]
Explode: [164462: handgrenade_throw.p3d,[1718.96,5509.69,5.53969],[0.34175,-0.295402,-0.014323]]
```

probably also effects https://github.com/acemod/ACE3/pull/9728

Reply:
>server player's grenade and rockets don't trigger on the 2nd client

Server player's grenade does trigger on client for me. I'm running the latest profiling branch (2.16.151879) on both clients, which makes me wonder if some fixes related to throwables did something?

Regardless, I can reproduce the behaviour for rockets, where it only explodes on the client the rocket was fired from.
I think the fix is easy enough: We add the mission EH on each client, but only the client where the projectile is local adds the `Explode` EH and triggers a global event.

>probably also effects https://github.com/acemod/ACE3/pull/9728

Yea, definitely does. It means that we can't do things server-side only.